### ISSUE TITLE
Add travis-CI testing

### ISSUE THREAD

**Issue by jonbinney** [iron-ox/trajectory_smoothing#18]
Once we have https://github.com/iron-ox/trajectory_smoothing/issues/17, I'd like to run the tests against all of our algorithms before merging new PRs. This will keep us from breaking old algorithms when we modify code in the core traj library.

**Comment by gavanderhoorn** [comment]
Since this is all Python, github actions may be a nice alternative.

`industrial_ci` also supports Github Actions, but plain actions already has nice support for Python, and seeing as this is more of a (plain) Python package than a ROS package, may be enough.

**Comment by jonbinney** [comment]
Good point. I've been meaning to try out github actions, so I'll give it a shot.

**Comment by gavanderhoorn** [comment]
I've used it [here](https://github.com/gavanderhoorn/dominh/tree/c191ce91eab57dba939f1814184658f48b022013/.github/workflows) as well.

Just for simple linting and `flake8`, but it's a start.
